Justerat css för cards

TobiasSvenssonDev opened the pull request:
Borde göra så att cards alltid har samma storlek och att innehållet i cards tar samma plats oavsett innehåll, dvs saknas en bild lämnas utrymme för bild. 
Fortfarande mer att fixa - tex går inte bilden ut på högersidan lika mycket som på vänster samt lite andra detaljer. Men detta är en bit på vägen.

TobiasSvenssonDev committed:
Justerat css för cards

Borde göra så att cards alltid har samma storlek och att innehållet i cards tar samma plats oavsett innehåll, dvs saknas en bild lämnas utrymme för bild

pontusnordstrom89 commented on mysite/staticfiles/css/globalstyle.css:
```diff
@@ -14,22 +14,59 @@ body {
-.col-sm-12 {
-    margin: 10%;
-    text-align: left, bottom;
 
```
innehåll mellan  {% block content %} och {% endblock %} bör stylas i respektive apps css fil. globalstyle hör till base_generic och bör bara hålla styling till header, nav och footer.

TobiasSvenssonDev replied:
Makes sense! Fortsatte nog utan att tänka då cards redan var styleade där.

pontusnordstrom89 commented on mysite/theStash/templates/theStash/index.html:
```diff
@@ -13,28 +13,29 @@
+    <div class="row card-main">
 
-        <h2><u>{{ item }}</u></h2>
+        <h2 class="cards-kategorier"><u>{{ item }}</u></h2>
```
var finns denna klassen? kan inte hitta den

TobiasSvenssonDev replied:
Yes, där var en plan! men sedan stötte jag på detaljer som min ocd inte kunde hantera och tiden sprang iväg men finlir på det ^^ klassen låg tidigare på en div om jag inte minns fel men flyttade den dit för att snygga till rubriken. Kom dock aldrig så långt så ej inskriven i något stylesheet ännu.
Men flyttar över allt med cards till rätt fil och kör igen när det är gjort.

TobiasSvenssonDev committed:
Ny pullrequest på samma

Har flyttat över till rätt css dokument (hoppas jag)

cards-category ej stylad men det kommer senare. Det finns en plan ^_^
